Drivers/c/AfecHwas/AfecHwas.c

```diff
@@ -0,0 +1,101 @@
+/**@file
+ * This file is part of the ARM BSP for the Test Environment.
```

Comment:
I missed this in the previous review. I think it would be better to state that this file is a part of DemoSat 2 validation use case to EDS TASTE Bridge project.